### PR TITLE
Added blog post about mentoring

### PR DESCRIPTION
**Pull request by CathyC93**

### CODE REVIEW

**Comment by KaraWaldemer** [comment]
I really like it! I’ll review it again in the morning when I’ve had some coffee but I don’t see any issues!

File: _posts/2020-07-28-mentoring-at-any-level.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## Mentoring at Any Level
+
+In my first software position post-college, I was a new developer in a program focused on teaching & training new developers how to build software in the “real world”. This opportunity was incredibly beneficial to the advancement of my skills and helped shape me into the developer I am today. Years later, I would go on to be a mentor for this same program, now teaching a new group of developers how to build software in the an industry environment. The main focuses of my job were to build products with my team and teach them what it means to write sustainable code while successfully implementing useful features. I found it incredibly exciting to watch the faces of those around me when something I was explaining or demonstrating really set in. Communicating technical concepts to others can be challenging on multiple levels, but that “eureka” moment makes it all worth it. This relationship with my team was mutually beneficial though, as I taught them what I knew, and they taught me my blind spots. They pointed out questions I hadn’t asked about technologies I used everyday, and they showed me the value of a completely fresh perspective.
```
Comment (patrickrgaffney) [inline _posts/2020-07-28-mentoring-at-any-level.md:14]: > ... how to build software in the an industry environment.

I think `the` can probably be dropped.

Comment (patrickrgaffney) [inline _posts/2020-07-28-mentoring-at-any-level.md:14]: Really love this paragraph, although perhaps I'm biased. 😎

Comment (CathyC93) [reply _posts/2020-07-28-mentoring-at-any-level.md]: Nice catch! I removed it

**Pull request merged**